src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_utils.py

```diff
@@ -6,6 +6,8 @@
 from azure.cli.core.util import CLIError
 from azure.cli.core.commands.parameters import get_resources_in_subscription
 
+from azure.mgmt.containerregistry.v2017_03_01.models import SkuTier
```

Comment:
@derekbekoe 
In this case, when user set CLI to be executed on an older version of API profile, will these commands be hidden from users?